VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -202,6 +202,7 @@ public void onEventMainThread(DiscussionCommentPostedEvent event) {
             final DiscussionThread discussionThread = discussionPostsAdapter.getItem(i);
             if (discussionThread.containsComment(event.getComment())) {
                 discussionThread.incrementCommentCount();
+                discussionThread.setRead(false);
```

Comment:
As I noted in my [comment][] on the method implementation, this should probably be done from the `incrementCommentCount()` method.

[comment]: https://github.com/edx/edx-app-android/pull/574/files#r52582108